### ISSUE TITLE
Impacts on finetuning?

### ISSUE THREAD

**Issue by lochuynh1412** [ahans30/goldfish-loss#1]
Hi, thanks for your great works.

I'm curious if this training approach will work for finetuning? Have you guys ever tried it to finetune pretrained models such as llama, mistral to mitigate memorization issue?

Bests,

**Comment by neelsjain** [comment]
Hello @lochuynh1412, that is a great question! We did not try finetuning on instruction data.

**Comment by neelsjain** [comment]
Hello @lochuynh1412, that is a great question!

We did not try fine-tuning on instruction data. However, the goldfish loss can be applied to IFT (instruction fine-tuning) data as well. When this loss is applied to the IFT data, we suspect that only the IFT data will not be memorized. However, we have not tried this experiment. Let us know if you have any more questions.
